Libraries/src/Amazon.Lambda.RuntimeSupport/Amazon.Lambda.RuntimeSupport.csproj

```diff
@@ -4,7 +4,7 @@
 
   <PropertyGroup>
     <TargetFrameworks>netstandard2.0;net5.0;net6.0;net8.0</TargetFrameworks>
-    <VersionPrefix>1.9.0</VersionPrefix>
+    <VersionPrefix>1.9.1</VersionPrefix>
```

Comment:
I was on the fence with "minor" - it's a new setting, but really offering a way to fallback to old behavior. Okay either way.

Reply:
I had similar thought but I didn't want to give the impression we added a feature since it is really just a way to turn off a feature.